compiler/repl-service/server/src/main/scala/com/digitalasset/daml/lf/ReplServiceMain.scala

```diff
@@ -36,12 +36,24 @@ object ReplServiceMain extends App {
       accessTokenFile: Option[Path],
       maxInboundMessageSize: Int,
       tlsConfig: Option[TlsConfiguration],
+      // optional so we can detect if both --static-time and --wall-clock-time are passed.
```

Comment:
Not really important here given that this is not a user-facing tool and I’m fairly confident we can write Haskell code that will not pass both but given that you already wrote it we might as well keep it I guess